### PR TITLE
feat: add accessibility labels

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:

[Accesibilidad en el panel de palabras](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=Accesibilidad%20en%20el%20panel%20de%20palabras)

## What & Why:

This PR adds aria labels and roles for accessibility purposes. 

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/constants/types.js [hunk 1]
```diff
@@ -6,24 +6,28 @@ export const LETTER_STATUS = {
...
     colorOrder: 2,
     icon: '🟨',
+    ariaLabel: (letter, t) => `${letter} ${t('ariaLabels.misplaced')}`,
   },
   nothing: {
     id: 'nothing',
     color: '#818384',
     colorOrder: 0,
     icon: '⬛️',
+    ariaLabel: (letter, t) => (!!letter ? letter : t('ariaLabels.empty')),
```
Comment (jpirazusta) [inline src/constants/types.js]:
```suggestion
    ariaLabel: (letter, t) => letter || t('ariaLabels.empty'),
```

**Review by jpirazusta — APPROVED** [review]

File: src/pages/Home/index.js [hunk 2]
```diff
@@ -51,27 +51,44 @@ const Home = () => {
...
-                  onClick={() => setLetterIndex(index)}
-                  disabled={round !== currentRound || gameEnded}
-                />
-              ))}
-            </>
-          </div>
-        ))}
+        {usersAttempts.map((attempt, round) => {
+          const attemptedWord = attempt.join('');
+          const wordAriaLabel = !!attemptedWord.length ? attemptedWord : t('ariaLabels.empty');
```
Comment (jeisux92) [inline src/pages/Home/index.js]: i think this should work

```suggestion
          const wordAriaLabel = attemptedWord ?? t('ariaLabels.empty');
```

**Review by jeisux92 — APPROVED** [review]
just a non-blocking comment

File: src/constants/types.js (hunk 1 above)
Comment (ManuViola77) [reply src/constants/types.js]: thanks!

File: src/pages/Home/index.js (hunk 2 above)
Comment (ManuViola77) [reply src/pages/Home/index.js]: thanks, I'll do the same as JP's suggestion: 

```suggestion
          const wordAriaLabel = attemptedWord || t('ariaLabels.empty');
```

Comment (jeisux92) [reply src/pages/Home/index.js]: cool!

**Pull request merged**